### ISSUE TITLE
Read Agora Documentation

### ISSUE THREAD

**Issue by B1s9l** [sopra-fs24-group07/sopra-fs24-group-07-server#50]
- [x] Decide on an agora "expert" as a group (@Monti934 )
- [x] Read agora documentation
- [x] implement a prototype locally

**Comment by Monti934** [comment]
Findings, learnings and problems:

- 99% frontend, only need backend for user or team information
- Voice Chat itself no problem with agora, good quality
- Breakout-rooms, join, leave and mute works
- user can connect via userID (needs to be unique)
- we can show the username (probably get it from backend via userId, or whatever is available)

- creating empty room? -> difficult, you join on create
- display created room? -> dont know yet if possible (thought about again, probably possible but is it elegant? (also live update of displayed rooms?))
- similar problem: display users in voice chat if i'm not in the voice chat

- idea: go back to idea from our meeting to automatically create breakout-rooms for each task
- display the rooms as buttons with which you can join the room
- then we can also automatically add TeamId to the roomId, so there are no duplicates between teams
- basically no user input to avoid errors and make it easy to use
